tuxemon/technique.py

```diff
@@ -214,7 +215,9 @@ def full_recharge(self) -> None:
     def reset_combat_counter(self) -> None:
         """Reset the combat counter."""
         self._combat_counter = 0
-
+    def keep_old_stats(self):
+        self.old_stats_data.append([mon.speed, mon.hp, mon.armour, mon.melee, mon.ranged, mon.dodge])
```

Comment:
this won't work because `mon` is not in this scope.  please playtest before asking for reviews.

Reply:
fixed

Reply:
ok i added a line to make mon work again

Reply:
i don't understand.  where is `mon` in this function?

Reply:
sorry, for whatever reason I typed the new line in but it didnt save the changes.